test/binding.gyp

```diff
@@ -39,6 +40,7 @@
       'dependencies': ["<!(node -p \"require('../').gyp\")"],
       'cflags': [ '-Werror', '-Wall', '-Wextra', '-Wpedantic', '-Wunused-parameter' ],
       'cflags_cc': [ '-Werror', '-Wall', '-Wextra', '-Wpedantic', '-Wunused-parameter' ],
+      'defines': ['NODE_ADDON_API_DISABLE_DEPRECATED']
```

Comment:
Does this mean we won't test the deprecated functions?

Reply:
Yes, it does mean that.